tests/integration/rdb.tcl

```diff
@@ -442,15 +442,21 @@ start_server [list overrides [list "dir" $server_path]] {
             restart_server 0 true false
             wait_done_loading r
 
-            assert_equal [lsort [r hgetall key]] "1 2 3 a b c"
+            # Never be sure when active-expire kicks in into action
+            wait_for_condition 10 10 {
```

Comment:
Is 100 milliseconds enough here? If this is flaky, I'd make it at least a few seconds. Just fyi.

Reply:
it isn't flaky. Just in case.